MAUI/Scheduler/cell-selection-view.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: Cell selection view in .NET MAUI Scheduler | Syncfusion
+description: Learn here all about how to customize the cell selection in Syncfusion .NET MAUI Scheduler (SfScheduler) control, its elements, and more.
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Cell Selection View in .NET MAUI Event Scheduler (SfScheduler).
+
+The Scheduler provides the capability to customize the selection of cell in all views using the cellselectionview.
+
+
+## Cell selection using Stroke
```

Comment:
Titles

## Stroke
## Background
## Stroke and Background
## Corner Radius
## Stoke Thickness
## Template

Reply:
Added the changes sir